### PR TITLE
feat: release 2.7

### PR DESCRIPTION
**Pull request by spacewander**
Signed-off-by: spacewander <spacewanderlzx@gmail.com>

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,6 +43,40 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 2.7.0
+
+### Change
+
+- change: check metadata_schema with check_schema like the other schema [#4381](https://github.com/apache/apisix/pull/4381)
+- change(echo): remove odd auth_value [#4055](https://github.com/apache/apisix/pull/4055)
+- fix(admin): correct the resources' count field and change its type to integer [#4385](https://github.com/apache/apisix/pull/4385)
```
Comment (tokers) [inline CHANGELOG.md:52]: This should be put under the `### Fix` ?

Comment (spacewander) [reply CHANGELOG.md:52]: This fix is a break change so I think it should be more noticeable.

**Review by gxthrj — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (tokers) [reply CHANGELOG.md:52]: OK, gotcha.

**Review by tokers — APPROVED** [review]

**Pull request merged**